spec/controllers/admin_controller_spec.rb

```diff
@@ -272,8 +272,9 @@
 
     include_context 'Admin user logged in'
 
+    # Reduced from 60 to 5 - just need to verify endpoint works, not stress test it
     before do
-      create_list(:manifestation, 60, language: 'ru', orig_lang: 'he')
+      create_list(:manifestation, 5, language: 'ru', orig_lang: 'he')
```

Comment:
I don't remember why I've used 60 here. Maybe there were some bug with pagination (so I need to have 50+ records). This is a quick stub just to ensure page is not failing.

It makes sense to add more checks and actually check logic